e2e.groovy

```diff
@@ -0,0 +1,57 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/Yuggupta27/ceph-csi'
+def ci_git_branch = 'e2e-centosci'
+def ref = "master"
+
+node('cico-workspace') {
+
+	stage('checkout ci repository') {
+		git url: "${ci_git_repo}",
+			branch: "${ci_git_branch}",
+			changelog: false
+	}
+
+	stage('reserve bare-metal machine') {
+		def firstAttempt = true
+		retry(30) {
+			if (!firstAttempt) {
+				sleep(time: 5, unit: "MINUTES")
+			}
+			firstAttempt = false
+			cico = sh(
+				script: "cico node get -f value -c hostname -c comment --retry-count ${cico_retries} --retry-interval ${cico_retry_interval}",
+				returnStdout: true
+			).trim().tokenize(' ')
+			env.CICO_NODE = "${cico[0]}.ci.centos.org"
+			env.CICO_SSID = "${cico[1]}"
+		}
+	}
+	try {
+		stage('prepare bare-metal machine') {
+			if ("${ghprbPullId}".length() != 0) {
+				ref = "pull/${ghprbPullId}/head"
+			}
+			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./prepare.sh ./multi-node-k8s.sh root@${CICO_NODE}:'
+			sh "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=/opt/build/go/src/github.com/ceph/ceph-csi --gitrepo=${ci_git_repo} --ref=master"
```

Comment:
ah, next error found:

`ci_git_repo` points to the repository with the test scripts. This should point to the repository with ceph-csi. You could use the environment variable `ghprbGhRepository` (set to `ceph/ceph-csi`, so need to prepend `https://github.com/`). use it similar to `ghprbPullId` above (check if set, use a default otherwise).

Reply:
https://jenkins-ceph-csi.apps.ci.centos.org/job/e2e_ygupta/13/ is a manual run, without a PR. It will checkout the master branch of the `ci_git_repo`, so it might get a little further.